frontend/src/components/CodeMirrorFileEditor.css

```
@@ -0,0 +1,112 @@
+.cm-file-editor__error {
+  display: inline-flex;
+  align-items: center;
+  min-height: 22px;
+  padding: 0 6px;
+  border: 1px solid var(--border, #333);
+  color: var(--text-muted, #888);
+}
+
+.cm-file-editor__status--unsaved {
+  color: var(--status-warning, #fbbf24);
+  border-color: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)